stable/newrelic-infrastructure/templates/daemonset.yaml

```
@@ -55,6 +55,10 @@ spec:
                   {{- end }}
             - name: "CLUSTER_NAME"
               value: {{ include "newrelic.cluster" . }}
+            {{- if .Values.disableKubeStateMetrics }}
+            - name: "DISABLE_KUBE_STATE_METRICS"
+              value: {{ .Values.kubeStateMetricsPodLabel | quote }}
```

Here is a typo here. You are using incorrect value, it should be
```suggestion
              value: {{ .Values.disableKubeStateMetrics | quote }}
```

---

Thanks for noticing, @sshishov. I'll fix it right away.

---

Fix at #21712